packages/vimeo-video-element/vimeo-video-element.js

```diff
@@ -101,6 +101,30 @@ class VimeoVideoElement extends (globalThis.HTMLElement ?? class {}) {
     this.#upgradeProperty('config');
   }
 
+  requestFullscreen() {
+    if (this.api?.requestFullscreen instanceof Function) {
+      return this.api.requestFullscreen();
+    }
```

Comment:
```suggestion
    return this.api?.requestFullscreen?.();
```

make all of them use optional chaining, that's sufficient and saves some bytes...